roles/satellite-clone/tasks/backup_check.yml

```diff
@@ -95,4 +95,4 @@
       remove some files so there is only one backup present. For an online backup, you need
       candlepin.dump, foreman.dump, and mongo_dump folder. For a standard (offline) backup,
       you will need mongo_data.tar.gz and pgsql_data.tar.gz.
```

Comment:
I liked the msg from your previous commit though which detailed all three scenarios clearly.